site/profile/manifests/users.pp

```diff
@@ -144,16 +144,31 @@
 
   $public_keys.each | Integer $index, String $sshkey | {
     $split = split($sshkey, ' ')
```

Comment:
splitting on `' '` would not work in case the `command=` option contains a space (space in the path)